wagtail/models/__init__.py

```diff
@@ -4487,8 +4335,9 @@ def approve(self, user=None, update=True, comment=""):
         self.log_state_change_action(user, "approve")
         if update:
             self.workflow_state.update(user=user)
+        specific_self = self.specific
         task_approved.send(
-            sender=self.specific.__class__, instance=self.specific, user=user
+            sender=specific_self.__class__, instance=specific_self, user=user
```

Comment:
Just curious: is there a specific (pun intended) reason for this change, other than readability? Since `specific` is a cached property, there is no noticeable performance difference between the two, is there?